### ISSUE TITLE
PluginManager.rezplugins_module_paths breaks with zipped Python

### ISSUE THREAD

**Issue by Ed-Whetstone-RT** [nerdvegas/rez#1108]
When I run the code below, I get an error stating that zipimporter object has no attribute `path`

I'm using Python 3.9.1 Embeddable and Rez 2.93 on Windows.  I think this is related to the embeddable python using a zipped directory to store the standard library.

If I use my system python rather than the packaged embeddable python, this code runs fine.

```
from rez.plugin_managers import RezPluginManager
RezPluginManager().rezplugins_module_paths
```

**Comment by maxnbk** [comment]
based on DeprecationWarnings I see in rez against python-3.8 on linux, I strongly suspect rez isn't ready for py3.9+ yet anyway. Might want to hold off even if this issue is solved.
Planning to go look at the DeprecationWarnings soon.

**Comment by nerdvegas** [comment]
Fixed by #1109
